Do not lick the supermatter, it is not made of candy.

TemporalOroboros opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
- Adds a special interaction for people attempting to lick the ~~candy~~supermatter.

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
It adds a little bit of extra fluff and humor.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Nanotransen would like to make an official notice to all stations equipped with an SME (SuperMatter Engine) Since the last batch of crystals were shipped we have noticed an increased workplace accident occurrence rate in the engineering department. The supermatter is not made of candy. Do not lick it.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

TemporalOroboros committed:
Do not lick the candy rock.

Fikou commented:
Literally 1984.

Tlaltecuhtli commented:
add emag effect to super matter

TemporalOroboros committed:
Merge branch 'master' into the-supermatter-is-not-made-of-candy

Iamgoofball commented on code/modules/power/supermatter/supermatter.dm:
```diff
@@ -923,7 +923,38 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
+	if(forehead)
+		user.visible_message(
+			"<span class='danger'>As [user]'s forehead bumps into [src], inducing a resonance... Everything goes silent before [user.p_their()] [forehead] flashes to ash!</span>",
+			"<span class='userdanger'>You feel your forhead bump into [src] and everything suddenly goes silent. As your head fills with ringing you come to realize that that was not a wise decision.</span>"
```
forehead

TemporalOroboros replied:
Yes

Mothblocks replied:
he meant theres a typo

TemporalOroboros replied:
Yes. I'll fix it in a bit.

TemporalOroboros replied:
Should be fixed in the next push.

Qustinnus commented on code/modules/power/supermatter/supermatter.dm:
```diff
@@ -923,7 +923,38 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
+		return
+
+	. = TRUE
+	if(user.combat_mode || (user.zone_selected != BODY_ZONE_PRECISE_MOUTH))
```
does this /really/ need to check combat mode?

TemporalOroboros replied:
It's probably not strictly necessary, but I might as well.

Cyberboss replied:
Let's not, please

Qustinnus replied:
No, we should be limiting the amount of combat mode checks.

Qustinnus replied:
(and be removing any that are left so we don't require modal interactions)

cacogen commented on code/modules/power/supermatter/supermatter.dm:
```diff
@@ -923,7 +923,38 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
+		dust_mob(user, cause = "hand")
+		return
+
+	var/obj/item/organ/tongue/lick = user.getorganslot(ORGAN_SLOT_TONGUE)
```
I would name this licking_tongue

TemporalOroboros replied:
Done. Should be in the next push.

TemporalOroboros committed:
Spelling fixes and var names. Also sounds.

- Fixes an instance of "forhead"
- Renames a var
- Adds supermatter sounds to the tongue being consumed

Megamaw commented:
man, the fact it doesn't even tell you what it tastes like after you lick it and die is _**heartbreaking.**_

good pr, though.

Cyberboss reviewed (dismissed):
~~I'm gonna veto the SM dusting individual bodyparts. It's not lore-friendly.~~

TemporalOroboros committed:
Merge branch 'master' into the-supermatter-is-not-made-of-candy

TemporalOroboros commented:
> 
> 
> I'm gonna veto the SM dusting individual bodyparts. It's not lore-friendly.

The supermatter already dusts individual bodyparts. Specifically arms and brains.

TemporalOroboros committed:
No more individual bodypart dusting + don't bite the supermatter either.
